lib/dashboard/grouped_organizations.rb

```diff
@@ -30,6 +30,9 @@ def collect_grouped_options
       groups = @organizations.group_by(&:type)
       options = ["Institution", "Provider", "Program", "Core"].map do |type|
         next unless groups[type].present?
+
+        # Sort organizations by name
+        groups[type].sort! { |lhs, rhs| lhs.name <=> rhs.name }
```

Comment:
If @organizations is an ActiveRecord::Relation, could we sort in the group_by instead of here?